ambari-common/src/main/python/resource_management/libraries/execution_command/execution_command.py

```diff
@@ -279,14 +280,20 @@ def get_group_list(self):
     Retrieve a list of user groups from command.json, i.e "group_list": "[\"hadoop\"]"
     :return: a list of groups
     """
-    return self.__get_value("stackSettings/group_list")
+    group_list = self.__get_value("stackSettings/group_list")
+    if not group_list:
+      group_list = "[]"
+    return group_list
 
   def get_user_list(self):
     """
     Retrieve a list of users from command.json, i.e "user_list": "[\"zookeeper\",\"ambari-qa\"]"
     :return: a list of users
     """
-    return self.__get_value("stackSettings/user_list")
+    user_list = self.__get_value("stackSettings/user_list")
```

Comment:
Now the command.json does not include stack_name, stack_version, user_list, group_list etc in stackSettings, i.e if I restart installed zookeeper service, the command.json has no such fields. We may discuss these fields and the locations later. At this time, just follow the old way.